internal/runtime/interface.go

```diff
@@ -161,9 +161,12 @@ type ClusterNamespace interface {
 }
 
 type Deployable interface {
+	// TODO remove when cli and orchestrator only use GetPackageRef().
 	// Returns a string to be compatible with the proto API.
 	GetPackageName() string // schema.PackageName
```

Comment:
This should already be removed in this change; the proto changes then come after.